Model/SofortPaymentMethod.php

```diff
@@ -167,4 +171,16 @@ public function getPublicFieldNames()
             ['additionalInfo', 'countryCode', 'logoType', 'deleteCanceledOrder', 'deleteFailedOrder']
         );
     }
+
+    /**
+     * @inheritdoc
+     *
+     * @return SepaCreditTransferPaymentMethod
+     *
+     * @since 1.0.1
+     */
+    public function getPostProcessingPaymentMethod()
+    {
```

Comment:
I got the idea today, i think you can add the Sepa CT config here. And not all the time in the get config, then its not added on the initial transaction of sofort. Please call me if you think i will not work.

Reply:
we always use the getConfig() when creating the backendService so I believe this wouldn't work, please have another look and if it is not clear we can have a call